run_surefire_rule_azure.sh

```diff
@@ -34,7 +34,7 @@ short_sha=${sha:0:7}
 modifiedslug_with_sha="${modifiedslug}-${short_sha}"
 
 # echo "================Cloning the project"
-bash $dir/clone-project.sh "$slug" "$sha"
+bash $dir/clone-project.sh "${modifiedslug_with_sha}=${modified_module}" "$sha"
```

Comment:
The modified_module variable would need to be defined before this line. Currently, it is defined in Line 97